modules/wordpress-com-block-editor-iframe.php

```diff
@@ -111,3 +111,64 @@ function jetpack_filter_salt( $salt, $scheme ) {
 
 	return $salt;
 }
+
+/**
+ * Enqueue the scripts for the WordPress.com block editor integration.
+ */
+function jetpack_enqueue_wpcom_block_editor_scripts() {
+	$suffix        = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+	$version       = gmdate( 'YW' );
```

Comment:
we should create an issue or some notice to come back and change this at some point. in my past I've regretted forgetting to remove date-based cache-busters.

Reply:
What problems did you have with versions based on dates? The reasoning behind this approach is to make sure that Jetpack users are getting the latest version of the scripts without requiring them to update the plugin (which also removes us the necessity to maintain this code every time we update the scripts).

Do you have in mind any other mechanism not involving date-based cache-busters that also maintains the dynamic versioning flexibility?

Reply:
the main consequences were that once the code reached a stable point people were needlessly re-downloading the script.

in this case a week isn't so bad but I just wanted to somehow mark this for a follow-up in something like six months to see if it's still needed

Reply:
Thanks for the context. It makes sense :) I created a [temporary card](https://github.com/orgs/Automattic/projects/88#card-20578307) in the House Tarly board which I'll convert to an issue once this PR is merged.

Reply:
> we should create an issue or some notice to come back and change this at some point. in my past I've regretted forgetting to remove date-based cache-busters.

@dmsnell you were totally right....... 

This week a version of the script was deployed which contained a bug. It was only noticed after the cache rolled over to the next day. So we reverted the faulty code and deployed it. But then people were confused why it wasn't fixed, because it took another 24 hours for the fix to be available to all users because of the cache.

So yeah, we definitely need to change this to something which invalidates automatically when the file is updated. 🤔